modules/RTC/TraceablePeerConnection.js

```diff
@@ -1481,15 +1481,15 @@ TraceablePeerConnection.prototype.addTrack = function(track, isInitiator = false
  * the renegotiation is required or <tt>false</tt> otherwise.
  */
 TraceablePeerConnection.prototype.addTrackUnmute = function(track) {
+    if (browser.usesUnifiedPlan()) {
```

Comment:
why is this call moved above the _assertTrackBelongs? a comment would be helpful

Reply:
I had to move this for the startWithVideoMuted=true case. Unmuting the video after join was failing this check. I moved the other removeTrackMute case for consistency. There is an additional check in TPCUtils removeTrackMute so it was safe to do so.